data/baibhav-tiwari.json

```
@@ -3,8 +3,26 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "**Web3** | **Web-developer** | **React** | **Open_Source**",
-
-  "tags": ["Open Source", "Javascript", "ReactJS", "Web3"],
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "ReactJS",
+    "Web3"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/BaibhavTiwari11"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/baibhav-tiwari"
+    },
+    {
+      "icon": "SiHashnode",
```

You might want to add the LinkedIn icon here, which should be `FaLinkedin`. 

```suggestion
      "icon": "FaLinkedin",
```